boards/arm/nucleo_l4r5zi/nucleo_l4r5zi_defconfig

```diff
@@ -14,6 +14,9 @@ CONFIG_BOARD_NUCLEO_L4R5ZI=y
 # 80MHz system clock in 'normal power' mode
 CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=80000000
 
+# clock for the LPTIMER
```

Comment:
Please put this in board Kconfig.defconfig under if lp timer condition

Reply:
we could just have in nucleo_xx_defconfig of the board:
 # enable lptimer and its clock source
CONFIG_STM32_LPTIM_TIMER=y
CONFIG_STM32_LPTIM_CLOCK_LSE=y

Reply:
choice for LSE or LSI clock for LPTIMER clock is now in the Kconfig.defconfig for the board, under STM32_LPTIM_TIMER condition:
`if STM32_LPTIM_TIMER

choice STM32_LPTIM_CLOCK
	default STM32_LPTIM_CLOCK_LSI

endchoice

endif #STM32_LPTIM_TIMER
`

Reply:
the application must now define the CONFIG_STM32_LPTIM_TIMER to enable the LPTIMER feature in **prj.conf**
> 
CONFIG_STM32_LPTIM_TIMER=y